kbs/config/sgx_default_qcnl.conf

```diff
@@ -2,15 +2,15 @@
   // *** ATTENTION : This file is in JSON format so the keys are case sensitive. Don't change them.
   
   //PCCS server address
-  "pccs_url": "https://localhost:8081/sgx/certification/v4/"
+  //"pccs_url": "https://localhost:8081/sgx/certification/v4/"
```

Comment:
Shouldn't the pccs_url be also set to `"https://api.trustedservices.intel.com/sgx/certification/v4/"` ?
Afaicr, sometime back attestation didn't work without setting pccs_url. I will retest this by tomorrow but wanted to mention it here for @mythi to comment.

Reply:
This should be fine if used for quote verification only. The QGS setting needs a valid pccs url